### PR TITLE
Trra 137/employee subtotals

### PR DESCRIPTION
**Pull request by aprigge**
-I've added a total line to the employee detail page
-Added check for only travel requests in the fiscal year for this page
-Actual expenditures replace approved amounts so with our test data where there are expenditures without approvals the approval employee total does not match what is in the column ($0 replaced with $x).

### CODE REVIEW

File: terra/models.py [hunk 1]
```diff
@@ -239,6 +239,11 @@ def total_funding(self):
     def vacation_days(self):
         return sum([v.vacation_days() for v in self.vacation_set.all()])
 
+    def total_days_out(self):
+        vacation_days = sum([v.vacation_days() for v in self.vacation_set.all()])
```
Comment (akohler) [inline terra/models.py]: @aprigge This replicates logic from the `vacation_days()` function above.  This should call that function instead.

File: terra/templates/terra/employee.html [hunk 2]
```diff
@@ -31,11 +31,13 @@ <h2>FY {{fiscal_year}} Travel Requests</h2>
...
                 <th scope="col">Actual Expenses</th>
                 <th scope="col">Days Out Working</th>
-                <th scope="col">Days Out Vacation</th>   
+                <th scope="col">Days Out Vacation</th>
+                <th scope="col">Days Out Total</th>    
             </tr>
         </thead>
         <tbody>
     {% for treq in employee.travelrequest_set.all %}
+    {% if treq.in_fiscal_year %}
```
Comment (akohler) [inline terra/templates/terra/employee.html]: Minor nit: indentation, this `if` should be inside the `for`, similar to line 58-59.

File: terra/templates/terra/employee.html [hunk 3]
```diff
@@ -48,9 +50,31 @@ <h2>FY {{fiscal_year}} Travel Requests</h2>
                 <td>{{treq.actual_expenses|currency}}</td>
                 <td>{{treq.days_ooo}}</td>
                 <td>{{treq.vacation_days}}</td>
+                <td>{{treq.total_days_out}}</td>
             </tr>
+    {% endif %}
```
Comment (akohler) [inline terra/templates/terra/employee.html]: Adjust indentation to match above.

**Comment by aprigge** [comment]
Thanks, @akohler. I made the requested changes. et me know if anything else should be changed.

**Pull request merged**